README.md

```diff
@@ -28,54 +28,41 @@ Require the library
 require 'graphql/batch'
 ```
 
-Define a GraphQL::Batch::Query derived class. Use group_key to specify which queries can be reduced into a batch query, and an execute class method which makes that batch query and passes the result to each individual query.
+Define a custom loader, which is initialized with arguments that are used for grouping and an execute method for doing the batch query.
```

Comment:
Right, I'll update it.  I decided the name `execute` sounded better for a query rather than a loader.